scss/_patterns_icons.scss

```
@@ -573,3 +563,1049 @@
     }
   }
 }
+
+// NEW
```

Same suggestion as before

```suggestion
// ADDITIONAL
// Generated with icon-svgs-to-mixins script
```